### ISSUE TITLE
Implement eSciDocs metadata standard (GFZ)

### ISSUE THREAD

**Issue by AlexDo1** [VForWaTer/metacatalog#295]
For the connection to GFZ, we need to implement their `eSciDocs` metadata standard, which is a combination of `ISO 19115` and `DataCite`.  
This way, we should have all necessary metadata on export for interoperability with GFZ dataservices.

Examples: [Template_escidoc.zip](https://github.com/VForWaTer/metacatalog/files/11000763/Template_escidoc.zip)

**Comment by mmaelicke** [comment]
I guess this is more a V-FOR-WaTer portal / backend issue and less of a metacatalog issue. The combination of ISO19115 and DataCite metadata seems to be a peculiarity of the GFZ portal and less a useful standards export format.

**Comment by AlexDo1** [comment]
Yes that's true, I just wanted to make sure we do not forget about that and I wanted to store the examples somewhere.

We can of course move the issue somewhere else.
